blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/processors/BatchWriter.java

```diff
@@ -122,10 +122,13 @@ public Boolean call() throws Exception {
                         singleBatchWriteCtx.stop();
                         bufferedMetrics.dec(batch.size());
 
-                        if (System.currentTimeMillis() - writeStartTime > timeout.toMillis()) {
+                        long now = System.currentTimeMillis();
+                        if ( now - writeStartTime > timeout.toMillis()) {
                             exceededScribeProcessingTime.mark();
-                            getLogger().error("Exceeded timeout " + timeout.toString() + " before persisting " +
-                                    "all metrics for batch " + batchId);
+                            getLogger().info(
+                                    String.format("Batch write time %d (ms) exceeded timeout %s before persisting " +
+                                            "all metrics for batch %d",
+                                            now - writeStartTime, timeout.toString(), batchId));
```

Comment:
I agree that its not an error. But somebody thought its really bad that we are exceeding that timeout. Any idea what this timeout actually is?